rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -99,14 +123,15 @@ public void startServer(Configuration conf, MetricRegistry metricsRegistry)
     }
     catch (BookKeeperInitializationException e) {
       log.error("Could not start BookKeeper daemon. Exception: ", e);
-      return;
+      throw Throwables.propagate(e);
     }
-
-    startThriftServer(conf, localBookKeeper);
   }
 
   void startServer(Configuration conf, BookKeeper bookKeeper, BookKeeperMetrics bookKeeperMetrics)
   {
+    conf = new Configuration(conf);
+    CacheConfig.setCacheDataEnabled(conf, false);
```

Comment:
same here

Reply:
Same explanation. We dont need to use CachingFileSystem ever inside BookKeeper or LDS